src/neuroglancer/ui/tool.ts

```diff
@@ -253,41 +258,65 @@ export class ToolBinder extends RefCounted {
   activate(key: string, inputEventMapBinder: InputEventMapBinder): Borrowed<Tool>|undefined {
```

Comment:
Now that `inputEventMapBinder` is no longer tied to the activation of a single tool, since it is saved by `debounceReactivate`, let's just make it a constructor parameter of `ToolBinder` instead.  The way you have it now works fine, but given that it will always be a fixed value I think it is clearer to make it a constructor parameter.

Reply:
How would that affect `makeToolActivationStatusMessage`? I don't fully understand what that does, but it modifies `activation.inputEventMapBinder` in a way that seems to depend on `activation`.

Reply:
`makeToolActivationStatusMessage` arranges for some additional code to be called whenever `inputEventMapBinder` is called, so that it can show the key bindings as part of the status message.

To be clear, we should retain the `inputEventMapBinder` member of `ToolActivation` --- you can just set it from the `inputEventMapBinder` set from the `ToolBinder` constructor.

Reply:
That makes sense, I have made that change.